dockstore-webservice/src/main/java/io/dockstore/webservice/core/Entry.java

```diff
@@ -219,8 +217,22 @@
     @CollectionTable(uniqueConstraints = @UniqueConstraint(name = "unique_paths", columnNames = { "entry_id", "filetype", "path" }))
     private Map<DescriptorLanguage.FileType, String> defaultPaths = new HashMap<>();
 
+    @ManyToMany(fetch = FetchType.EAGER)
```

Comment:
Might be explained later or I might be missing context
Is this eager for a particular reason or is this eager to essentially match the current behaviour of the getter and setter?
i.e. would a valid follow-up ticket be to make fileformats lazy loaded

Reply:
Yeah it's eager to match how it was being returned already, so could be good to try to make this lazy instead.

Reply:
Sounds good, then please make a ticket for lazy file formats when this is merged with this context